config/operator/deployment/deployment.yaml

```diff
@@ -69,6 +69,12 @@ spec:
           value: quay.io/openstack-k8s-operators/openstack-operator:latest
         - name: ENABLE_WEBHOOKS
           value: false
+        - name: LEASE_DURATION
```

Comment:
@hjensas I'm inclined to decrease these back towards the defaults. Perhaps 30/20/5 would be a reasonable default for us. And then folks who want to try out OpenShift cluster high availability mode have that option which we could document. The defaults seem to be https://pkg.go.dev/k8s.io/client-go/tools/leaderelection 15/10/2 I think.

Reply:
ok, yeah my eyes when *blink blink* a couple of times when I saw how much difference there was the [openshift/library-go](https://github.com/openshift/library-go/blob/4b9033d00d37b88393f837a88ff541a56fd13621/pkg/config/leaderelection/leaderelection.go#L96-L109) compared to https://pkg.go.dev/k8s.io/client-go/tools/leaderelection. 

I also noticed in my testing that if the operators where already running - the "old" operator would terminate and then the new one would wait for over a minute to get the lock. It made me think, it would be good if controlled termination of a leader released/deleted the lease ... 

Let's try it a bit lower, 30/20/5 sounds good. I can test that a bit, and if I need to patch the csv in labs to bump it a bit more that is ok.

Reply:
> ok, yeah my eyes when _blink blink_ a couple of times when I saw how much difference there was the [openshift/library-go](https://github.com/openshift/library-go/blob/4b9033d00d37b88393f837a88ff541a56fd13621/pkg/config/leaderelection/leaderelection.go#L96-L109) compared to https://pkg.go.dev/k8s.io/client-go/tools/leaderelection.

thank you for that link. I was wondering how lose values were chosen